interfaces/builtin/content.go

```diff
@@ -119,18 +119,20 @@ func (iface *contentInterface) SanitizePlug(plug *snap.PlugInfo) error {
 
 // path is an internal helper that extract the "read" and "write" attribute
 // of the slot
-func (iface *contentInterface) path(attrs map[string]interface{}, name string) []string {
+func (iface *contentInterface) path(attrs interfaces.AttrGetter, name string) []string {
```

Comment:
Per comments in the other PR, I think this should be Attributer or similar, so it doesn't feel like it has an AttrGet method, or that it takes and attr as a value, or something similarly nitpicky. :)

Reply:
Yes, I fixed the name already in the other PR (which is a prerequisite to this PR), so it will become 'Attrer' once it lands.